configuration/rules-dsl.md

```diff
@@ -178,6 +179,27 @@ then
 end
 ```
 
+### Thing Triggers
+
+You can listen on status updates or on status changes (an update might leave the status unchanged).
+You can decide whether you want to catch only a specific status or any.
+Here is the syntax for all these cases (parts in square brackets are optional):
+
+```java
+Thing <thingUID> received update [<status>]
+Thing <thingUID> changed [from <status>] [to <status>]
+```
+
+The status used in the trigger and the script is a string (no quotes).
+You can find all the possible values for status from [Thing Status]({{base}}/concepts/things.html).
+And refer to [Thing Status Action]({{base}}/addons/actions.html) to find how to get thing status in the script.
+In the trigger, thingUID is the identifier assigned by openHAB to the thing you add.
+You can find it from PaperUI or from Karaf remote console.
+For example, one z-wave device can be "zwave:device:c5155aa4:node14".
+
+> Note: You need to use quote around thingUID if it contains special characters such as ':'.
+> The parser will not recognize the whole text correctly without quotes.
```

Comment:
@kaikreuzer makes sense for things defined via config files, my things do not have : in their name and would naturally work without quotes.